core/db/migrate/20210716104141_add_index_on_name_parent_id_and_taxonomy_id_on_spree_taxons.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexOnNameParentIdAndTaxonomyIdOnSpreeTaxons < ActiveRecord::Migration[5.2]
+  def change
+    unless index_exists?(:spree_taxons, [:name, :parent_id, :taxonomy_id], unique: true)
+      remove_index :spree_taxons, [:name, :parent_id, :taxonomy_id] if index_exists?(:spree_taxons, [:name, :parent_id, :taxonomy_id])
+      add_index :spree_taxons, [:name, :parent_id, :taxonomy_id], unique: true
```

Comment:
@szymoniwacz before adding these unique indexes let's check if there are any duplicates and remove them